### PR TITLE
denoiseprofile documentation

### PR DESCRIPTION
**Pull request by rawfiner**
This PR updates the denoise profiled documentation to document the changes made since last version.
I am not a native speaker, as such I would like to have some feedback, to fix mistakes and make the explanations clearer :-)

### CODE REVIEW

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 1]
```diff
@@ -86,17 +96,29 @@
     <sect5>
       <title>mode</title>
       <para>
-        This module can eliminate noise with two different core algorithms. <quote>non-local
-        means</quote> can tackle efficiently luma (lightness) noise whereas it is less efficient
-        with chroma (color noise). <quote>wavelet</quote> is particularly efficient at
-        eliminating chroma (color) noise but can also give good results for luma noise. It also
-        lets you adjust the force of the denoising depending on the noise coarseness. If needed
+        This module can eliminate noise with two different core algorithms. Both <quote>non-local
```
Comment (jenshannoschwalm) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: by two different

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 2]
```diff
@@ -86,17 +96,29 @@
...
+        means</quote> and <quote>wavelet</quote> can tackle efficiently luma (lightness) noise
+        and chroma (color noise).  <quote>wavelet</quote>  mode also lets you adjust the force
```
Comment (jenshannoschwalm) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: and chroma (color) noise.

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 3]
```diff
@@ -86,17 +96,29 @@
...
+        of the denoising depending on the noise coarseness. If needed
         you can apply two instances of this module (see <xref linkend="multiple_instances"/>):
         one instance to tackle luma noise with blend mode <quote>lightness</quote> or <quote>HSV
         lightness</quote>, and another one to tackle chroma noise with blend mode
         <quote>color</quote> or <quote>HSV color </quote>. An example of the use of two
         instances with blending modes is available within the presets of this module. For more
         information on blend modes have a look at <xref linkend="blending_operators"/>.
+	The module also offers one automatic mode for each algorithm. Automatic modes allow
```
Comment (jenshannoschwalm) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: an automatic

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 4]
```diff
@@ -86,17 +96,29 @@
...
+	All sliders of this module can take values higher than their bounds if needed using Right-click.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>whitebalance-adaptive transform</title>
+      <para>
+	As white-balance amplifies the RGB channels differently, they exhibit
+	different noise levels. This checkbox makes the algorithm adaptive to white balance.
+	This option should be disabled if you used a first instance with a color blend mode.
```
Comment (chrik5) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: if you use a first

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 5]
```diff
@@ -115,11 +137,34 @@
       <title>search radius</title>
       <para>
         This slider is only available if mode <quote>non-local means</quote> is selected. It
-        controls the how far from a pixel the algorithm will try to find similar patches. The
+        controls how far from a pixel the algorithm will try to find similar patches. The
         processing time is hugely impacted by this parameter: it depends on the square of the
         parameter. A lower value will make execution faster, a higher value will make it slower.
         Increasing the value can give better results for very noisy images when coarse grain
-        noise is visible.
+        noise is visible, but you should better use the scattering slider instead.
```
Comment (chrik5) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: sentence could placed after: ... try to find similar patches. Increasing the value ...

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 6]
```diff
@@ -115,11 +137,34 @@
...
+    </sect5>
+    
+    <sect5>
+      <title>central pixel weight</title>
+      <para>
+	This slider is only available if mode <quote>non-local means</quote> or <quote>non-local means
+	auto</quote> is selected. It controls the amount of details which should be preserved by the
+	algorithm. It can be used as a way to control the amount of luma noise smoothing: giving a big
+	value to this slider will result mostly in chroma noise smoothing with little smoothing of chroma
+	noise. This slider has no effect if patch-size is set to 0.
```
Comment (chrik5) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: Doest this makes sense: ...chroma noise smoothing with little smoothing of chroma noise...

File: doc/usermanual/darkroom/modules/correction/denoise_profiled.xml [hunk 7]
```diff
@@ -175,6 +220,37 @@
...
+
+    <sect5>
+      <title>adjust autoset parameters</title>
+      <para>
+	This option is available in <quote>auto</quote> modes. In these modes, darktable tries to
+	infer denoising parameters from the camera profile. Yet, the automatically infered parameters
+	may be not optimal depending on your image. If your picture is very underexposed, you will have to
+	increase this parameter to get a proper denoising. This parameter should reflect the amplification
+	you add to your picture: if you add 1EV of exposure, the signal is multiplied by 2, so this
+	parameter should be set to 2.
```
Comment (chrik5) [inline doc/usermanual/darkroom/modules/correction/denoise_profiled.xml]: In these modes, darktable tries to derive denoising parameters from the camera profile. Depending on your image the automatically derived parameters may be not optimal. E.g. if your image is heavily underexposed and you lifted the exposure, you will have to increase this parameter to get a proper denoising. This parameter should reflect the amplification you add to your picture: if you add 1EV of exposure, the signal is multiplied by 2, so this parameter should be set to 2.

**Comment by rawfiner** [comment]
Many thanks to both of you for your comments. I will update the PR asap :-)

**Comment by rawfiner** [comment]
I have just updated the text (and screenshots). Thanks for the nice suggestions for the english writing, and for spotting my mistakes :-)

**Comment by TurboGit** [comment]
Thanks!

**Pull request merged**